pageStyles/sigmaker.styles.ts

```diff
@@ -72,6 +72,14 @@ const StyledInputField = styled(TextField)({
   width: '25ch'
 })
 
+const StyledGrid = styled(Grid)({
+  rowGap: 3
+})
+
+const StyledFormControl = styled(FormControl)({
+  my: 3
```

Comment:
I don't think we can use shorthand like this in `styled` - see "`sx` provides more shortcuts than `styled`" here: https://mui.com/system/styled/